### DISCUSSION TITLE
Is their a way to access properties of a wallpaper and store it as variable in autohotkey?

### DISCUSSION THREAD

**Discussion by Ucef27** [rocksdanister/lively#386] [Q&A] [answered]
I am trying to make some autohotkey functions I wrote a little bit better. Also can you access and store the volume?

I know I'm asking a lot of questions but I really love this software and am trying to get the most out of it. Thanks for all the help.

**Comment by rocksdanister** [comment, marked as answer]
Its just json files, you can read it:
https://github.com/rocksdanister/lively/wiki/Web-Guide-IV-:-Interaction#lively-properties

Files can be found:
`C:\Users\<UserName>\AppData\Local\Lively Wallpaper\Library\SaveData\wpdata\<wallpaper_random_folder>\<monitor_id>\`
<wallpaper_random_folder>: The foldername under `Library\wallpapers` for .zip wallpapers or `Library\wptmp` for reference type.

**Reply by Ucef27** [reply]
Thanks, and is their a way to access the volume?

**Reply by rocksdanister** [reply]
No.

**Reply by Ucef27** [reply]
Ok, thanks a lot!
